etl/pad/raw/skills/behaviors/active_behaviors.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Dict, List, Literal, NamedTuple, Optional, TYPE_CHECKING, Union
```

Comment:
is there a reason for hierarchically categorizing actives, as opposed storing them flat (e.g one class each for fixed damage, team damage, gravity... etc for ASBDamage)? I found ilmina's way of subgrouping these things to be slightly weird (i dont expect to look for fujin / attr under "Void")